channels.yaml

```diff
@@ -133,7 +133,15 @@ releases:
       etcd-arg:
         type: array
     agentArgs: *agentArgs-v1
-  - version: v1.22.4+k3s1
+  - version: v1.21.8-rc1+k3s1
+    minChannelServerVersion: v2.6.0-alpha1
+    maxChannelServerVersion: v2.6.99
+    serverArgs: &serverArgs-v2
+      <<: *serverArgs-v1
+      etcd-arg:
+        type: array
```

Comment:
I think we want to pull from `*serverArgs-v2` and then not include the `&serverArgs-v2` on line 139 here since we don't have any new args on this release and this new etcd-arg should be covered in the 1.21.7 section above.